package.json

```diff
@@ -46,23 +47,22 @@
     "@types/jest": "^24.0.11",
     "@types/js-yaml": "^3.12.1",
     "@types/json-pointer": "^1.0.30",
-    "@types/jsonpath": "^0.2.0",
     "@types/lodash": "4.14.123",
     "@types/md5-file": "^4.0.0",
     "@types/node": "^10.12.18",
     "@types/request": "^2.48.1",
     "@types/swagger-parser": "^4.0.3",
     "@types/uuid": "^3.4.4",
     "@types/yargs": "^11.1.2",
-    "ts-jest": "^24.0.2",
     "jest": "^24.7.1",
     "jest-junit": "^6.3.0",
     "prettier": "^1.18.2",
     "should": "^13.2.3",
+    "ts-jest": "^24.0.2",
     "tslint": "^5.17.0",
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.0.1",
-    "typescript": "~3.4.5"
+    "typescript": "^3.5.3"
```

Comment:
Why do we change to caret ranges? Does typescript minor version change have breaking change?

Reply:
We would like to automatically update to 3.6 when it's available. '^' is a kind of standard and default behavior. We introduce '~' only because we couldn't use `3.5` before but I fixed these issues.